store/iscn.ts

```diff
@@ -19,23 +20,28 @@ export default class ISCN extends VuexModule {
     [key: string]: ISCNRecordWithID;
  } = {};
 
-  recordsByIPLD: {
-    [key: string]: ISCNRecordWithID;
-  } = {};
+  records: ISCNRecordWithID[] = [];
+
+  isLoading = false;
 
   @Mutation
   setRecords(records: ISCNRecord[]) {
     records.forEach((r) => {
       const id = r.data['@id'] as string;
       Vue.set(this.recordsById, id, { id, ...r })
-      Vue.set(this.recordsByIPLD, r.ipld, { id, ...r })
+      this.records.push({ id, ...r })
     })
   }
```

Comment:
Since this function involves `.push()` now I don't think `setRecords` is a good name, maybe `addRecords` or `appendRecords` is better